ext-apiserver/cloud/google/pods.go

```diff
@@ -60,19 +62,49 @@ func getBase64(file string) string {
 }
 
 func getApiServerCerts() (*caCertParams, error) {
-	// TODO: remove the need to use apiserver-boot to generate the certificate.
-	os.RemoveAll("./config")
-	err := run("apiserver-boot", "build", "config",
-		"--name", "clusterapi", "--namespace", "default",
-		"--image", machineControllerImage)
+	const name = "clusterapi"
+	const namespace = corev1.NamespaceDefault
+	configDir, err := ioutil.TempDir("", "cert")
 	if err != nil {
 		return nil, err
 	}
+	defer os.RemoveAll(configDir) // clean up
+
+	if err := run("openssl", "req", "-x509",
```

Comment:
Why not using `"k8s.io/client-go/util/cert"` & `"k8s.io/client-go/util/cert/triple"` for certificate generation?

Reply:
Agreed. I don't want to exchange one external binary dependency for another.

Reply:
+1

Reply:
If the library provides the same functionality, +1.

Reply:
```go
//import "k8s.io/client-go/util/cert/triple"
caKeyPair, err := triple.NewCA("root-ca.example.com")
if err != nil {
	return fmt.Errorf("failed to create root-ca: %v", err)
}

apiserverKeypair, err := triple.NewServerKeyPair(caKeyPair, "apiserver.example.com", "kubernetes", "default", "cluster.local", []string{"10.10.10.1"}, []string{"some-other-hostname"})
if err != nil {
	return fmt.Errorf("failed to create apiserver key pair: %v", err)
}

//to get it pem encoded as byte slice:
// import "k8s.io/client-go/util/cert"
b1 := cert.EncodePrivateKeyPEM(apiserverKeypair.Key)
b2 := cert.EncodeCertPEM(apiserverKeypair.Cert)
```